test/component/uk/gov/hmrc/individualsemploymentsapi/stubs/BaseSpec.scala

```diff
@@ -31,10 +31,9 @@ trait BaseSpec
     extends FeatureSpec with BeforeAndAfterAll with BeforeAndAfterEach with Matchers with GuiceOneServerPerSuite
     with GivenWhenThen {
 
-  override lazy val port = 9000
   implicit override lazy val app: Application = GuiceApplicationBuilder()
     .configure(
-      "auditing.enabled"                                       -> false,
+      "auditing.enabled"                                 -> false,
```

Comment:
looks unintentional, probably some auto merge reformatting?

Reply:
If you pull the branch and reformat it does not pick this up. Looks correct in IntelliJ